brownie/network/contract.py

```diff
@@ -1208,7 +1221,23 @@ def get_solc_version(cls, compiler_str: str, address: str) -> Version:
 
         return version
 
-    def set_alias(self, alias: Optional[str]) -> None:
+    @classmethod
+    def remove_deployment(
+        cls, address: str, alias: str = None
+    ) -> Tuple[Optional[Dict], Optional[Dict]]:
+        """
+        Removes this contract from the internal deployments db.
+
+        Arguments
+        ---------
+        address: str
+            An address to apply
+        alias: str | None
+            An alias to apply
+        """
+        return _remove_deployment(address, alias)
+
+    def set_alias(self, alias: Optional[str], persist: bool = True) -> None:
```

Comment:
check if alias can be used within the same process if not persisted

Reply:
it's still there as a field on the contract object regardless of the persist flag.